contents/scalability/scalable-development.md

```diff
@@ -1,9 +1,9 @@
 # Scalable Development - An Introduction
 
-Authors: Loh Jia Shun Kenneth
+Authors: Loh Jia Shun Kenneth, Vivek Lakshmanan
 
 ## Scalable Development
-When Pokémon Go first launched in July 6, the heavy traffic from users caused its servers to crash. However, as of 2017, Google handles at least 3.5 billion searches daily, Whatsapp handles at least 10 billion messages a day, and Facebook hosts 1.2 billion daily active users. What makes these companies different?
+When Pokémon Go first launched in 2016, the heavy traffic from users caused its servers to crash as the server was built to handle an expected maximum of five times the storage size instead of the reality which was fifty times the initial estimate. However, as of 2017, Google handles at least 3.5 billion searches daily, Whatsapp handles at least 10 billion messages a day, and Facebook hosts 1.2 billion daily active users. What makes these companies different?
```

Comment:
> server was built to handle an expected maximum of five times the storage size instead of the reality which was fifty times the initial estimate.

What does "storage size" refer to? I can't quite get what this sentence is saying because it talks about "five times the storage size" vs "fifty times the initial estimate", which seems to be comparing about different things.